zebrad/src/components/sync.rs

```diff
@@ -204,6 +205,83 @@ const SYNC_RESTART_DELAY: Duration = Duration::from_secs(67);
 /// a denial of service on those peers.
 const GENESIS_TIMEOUT_RETRY: Duration = Duration::from_secs(5);
 
+/// Sync configuration section.
```

Comment:
Optional: This module is already over 1000 lines long, would you mind putting the config in a `sync/config.rs` submodule?